### ISSUE TITLE
Report on supported file types in backend endpoint

### ISSUE THREAD

**Issue by shuttle1987** [persephone-tools/persephone-web-API#140]
In #139 we added in an endpoint to let the API users know what files would be accepted, this closed #6 but in a nasty manner in order to unblock frontend work. This is currently hardcoded and should be generated from what's actually accepted both by the flask uploads and the backend.

**Comment by shuttle1987** [comment]
This requires a new feature in the Persephone library to support this.

**Comment by shuttle1987** [comment]
This is a duplicate of #135
